docs/guides/use_custom_images.md

```diff
@@ -0,0 +1,23 @@
+To use custom images when building a deployment you will have to use a certain environment variables. There are total 5 docker images used in a deployment setup.
+
+1. `valory/open-autonomy` 
+   
+    This image contains all the required python packages and a virtual environment required to deploy an agent. When building a deployment, the deployment builder will use `valory/open-autonomy` as the name and the current version of the framework as the version tag. If you want to change that you can change the name and the version of the image using `AUTONOMY_IMAGE_NAME` and `AUTONOMY_IMAGE_VERSION` environment variables respectively.
+   
+2. `valory/oar-{agent}` 
+    
+    This image contains an agent package and an deployment environment for the same agent. This image uses the open `valory/open-autonomy` as the base. You cannot change the name of the image but you can overwrite the version tag using the `--image-version` flag when building the deployment.
+
+3. `valory/open-autonomy-tendermint`
+   
+   This image contains the deployment setup for a `tendermint` node. When building a deployment, the deployment builder will use `valory/open-autonomy-tendermint` as the name and the current version of the framework as the version tag. If you want to change that you can change the name and the version of the image using `TENDERMINT_IMAGE_NAME` and `TENDERMINT_IMAGE_VERSION` environment variables respectively.
+
+4. `valory/open-autonomy-hardhat` 
+   
+   This image comes with the autonolas registry contracts pre deployed. This image can be only used when you're working with the `dev` mode. When building a deployment, the deployment builder will use `valory/open-autonomy-hardhat` as the name and `latest` as the version tag. If you want to change that you can change the name and the version of the image using `HARDHAT_IMAGE_NAME` and `HARDHAT_IMAGE_VERSION` environment variables respectively.
+
+   If you want to include custom contracts in this image read [here](https://github.com/valory-xyz/autonolas-registries/blob/main/docs/running_with_custom_contracts.md) on how to do it.
```

Comment:
@jmoreira-valory please advise as to path this will have on docs.autonolas.network

Reply:
I would say this belong to 'advanced_reference', rather than 'guides':

https://docs.autonolas.network/open-autonomy/advanced_reference/use_custom_images/

Reply:
Addressed [853bacf](https://github.com/valory-xyz/open-autonomy/pull/1757/commits/853bacf9dbf81b08f3e7751dea443aecbf49bc38)